sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5176,3 +5176,10 @@ def test_issue_6134():
      0                              0                   \
 """)
     assert upretty(e) == ucode_str
+
+def test_issue_9877():
+    from sympy.abc import a, b, c, x, y, z
+    
+    ucode_str = u('{x} u"\u2229" {y} u"\u2229" ({z} u"\u005C" {1,2})')
```

Comment:
Maybe it will be better to use unicode characters. Very difficult to read otherwise.